scripts/images/convert_qcow2_ova.py

```diff
@@ -440,6 +450,10 @@ def main(argv):
             'rhPassword',
             'osPassword'
         ])
+        if len(argv) < 14:
```

Comment:
will have to find a better way with other libraries to check the required args, will take that as a separate task,!

Reply:
Agree.. this is ugly.. We should switch to `argparse` going forward